src/vos/tests/evt_ctl.c

```diff
@@ -78,6 +78,14 @@ static daos_handle_t		ts_toh;
 #define EVT_SEP_EXT		'-'
 #define EVT_SEP_EPC		'@'
 
+/* Data sizes */
+#define D_1K_SIZE		1024
+#define D_16K_SIZE		(16 * 1024)
```

Comment:
Should we consider the non standard sizes for example 16392 bytes?

Reply:
Same as above. Some tests cover different sizes. This test covers some incremental values.

Reply:
I'm more worried about testing various overlaps and multiples of tree node size and such.  These tests are just more like out of memory.   How long does test take now?

Reply:
DAOS-2339 should address testing various overlaps and multiples of tree node size